diff --git i/config.def.h w/config.def.h
index 1c0b587..ab5d1bb 100644
--- i/config.def.h
+++ w/config.def.h
@@ -7,19 +7,18 @@ static const int showbar            = 1;        /* 0 means no bar */
 static const int topbar             = 1;        /* 0 means bottom bar */
 static const char *fonts[]          = { "monospace:size=10" };
 static const char dmenufont[]       = "monospace:size=10";
-static const char col_gray1[]       = "#222222";
-static const char col_gray2[]       = "#444444";
-static const char col_gray3[]       = "#bbbbbb";
-static const char col_gray4[]       = "#eeeeee";
-static const char col_cyan[]        = "#005577";
+static const char col_pink[]   	    = "#FF33B1";
+static const char col_purple[]      = "#D716FE";
+static const char col_blue[]        = "#1658FE";
+static const char col_yellow[]      = "#FFE633";
 static const char *colors[][3]      = {
 	/*               fg         bg         border   */
-	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
-	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
+	[SchemeNorm] = { col_blue, col_pink, col_purple },
+	[SchemeSel]  = { col_blue, col_yellow,  col_pink  },
 };
 
 /* tagging */
-static const char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
+static const char *tags[] = { "Term", "Firefox", "Chromium", "IntelJ", "Vim", "Other" };
 
 static const Rule rules[] = {
 	/* xprop(1):
@@ -27,10 +26,12 @@ static const Rule rules[] = {
 	 *	WM_NAME(STRING) = title
 	 */
 	/* class      instance    title       tags mask     isfloating   monitor */
-	{ "Gimp",     NULL,       NULL,       0,            1,           -1 },
-	{ "Firefox",  NULL,       NULL,       1 << 8,       0,           -1 },
-};
-
+	{ "st-256color",  "st-256color",       NULL,       1 << 0,       0,           -1 },
+	{ "Firefox", NULL, NULL,       1 << 1,       0,           -1 },
+	{ "Chromium",  "chromium",       NULL,       1 << 2,       0,           -1 },
+	{ "jetbrains-idea", NULL, NULL,       1 << 3,       1,           -1 },
+	{ "vim",     "st-256color",    NULL,       1 << 4,       0,           -1 }
+}; 
 /* layout(s) */
 static const float mfact     = 0.55; /* factor of master area size [0.05..0.95] */
 static const int nmaster     = 1;    /* number of clients in master area */
@@ -44,7 +45,7 @@ static const Layout layouts[] = {
 };
 
 /* key definitions */
-#define MODKEY Mod1Mask
+#define MODKEY Mod4Mask
 #define TAGKEYS(KEY,TAG) \
 	{ MODKEY,                       KEY,      view,           {.ui = 1 << TAG} }, \
 	{ MODKEY|ControlMask,           KEY,      toggleview,     {.ui = 1 << TAG} }, \
@@ -56,23 +57,38 @@ static const Layout layouts[] = {
 
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
+static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_blue, "-nf", col_pink, "-sb", col_yellow, "-sf", col_pink, NULL };
+static const char *vimcmd[]  = { "st", "-c", "vim", "vim", NULL };
 static const char *termcmd[]  = { "st", NULL };
+static const char *voldown[]  = { "amixer", "set", "Master", "10%-", NULL };
+static const char *volup[]  = { "amixer", "set", "Master", "10%+", NULL };
+static const char *voltoggle[] = { "amixer", "set", "Master", "toggle", NULL };
+static const char *firefox[] = { "firefox", NULL };
+static const char *chromium[] = { "chromium", NULL };
+static const char *intellij[] = { "intellij-idea-ultimate-edition", NULL };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
 	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
+	{ MODKEY,                       XK_v,      spawn,          {.v = vimcmd } },
 	{ MODKEY|ShiftMask,             XK_Return, spawn,          {.v = termcmd } },
-	{ MODKEY,                       XK_b,      togglebar,      {0} },
+	{ MODKEY,			XK_F1,	   spawn, 	   {.v = voldown } },
+        { MODKEY,                       XK_F2,     spawn,          {.v = volup } },
+        { MODKEY,			XK_F3,     spawn,	   {.v = voltoggle } },
+	{ MODKEY,                       XK_t,      spawn,          {.v = termcmd } },
+        { MODKEY,                       XK_f,      spawn,          {.v = firefox } },
+	{ MODKEY,                       XK_c,      spawn,          {.v = chromium } },
+        { MODKEY,                       XK_i,      spawn,          {.v = intellij } },
+        { MODKEY,                       XK_b,      togglebar,      {0} },
 	{ MODKEY,                       XK_j,      focusstack,     {.i = +1 } },
 	{ MODKEY,                       XK_k,      focusstack,     {.i = -1 } },
-	{ MODKEY,                       XK_i,      incnmaster,     {.i = +1 } },
+        { MODKEY,                       XK_I,      incnmaster,     {.i = +1 } },
 	{ MODKEY,                       XK_d,      incnmaster,     {.i = -1 } },
 	{ MODKEY,                       XK_h,      setmfact,       {.f = -0.05} },
 	{ MODKEY,                       XK_l,      setmfact,       {.f = +0.05} },
 	{ MODKEY,                       XK_Return, zoom,           {0} },
 	{ MODKEY,                       XK_Tab,    view,           {0} },
-	{ MODKEY|ShiftMask,             XK_c,      killclient,     {0} },
+	{ MODKEY,                       XK_q,      killclient,     {0} },
 	{ MODKEY,                       XK_t,      setlayout,      {.v = &layouts[0]} },
 	{ MODKEY,                       XK_f,      setlayout,      {.v = &layouts[1]} },
 	{ MODKEY,                       XK_m,      setlayout,      {.v = &layouts[2]} },
@@ -90,10 +106,7 @@ static Key keys[] = {
 	TAGKEYS(                        XK_4,                      3)
 	TAGKEYS(                        XK_5,                      4)
 	TAGKEYS(                        XK_6,                      5)
-	TAGKEYS(                        XK_7,                      6)
-	TAGKEYS(                        XK_8,                      7)
-	TAGKEYS(                        XK_9,                      8)
-	{ MODKEY|ShiftMask,             XK_q,      quit,           {0} },
+	{ MODKEY|ShiftMask,               XK_q,      quit,           {0} },
 };
 
 /* button definitions */
diff --git i/config.mk w/config.mk
index 25e2685..87e8299 100644
--- i/config.mk
+++ w/config.mk
@@ -4,11 +4,11 @@ VERSION = 6.1
 # Customize below to fit your system
 
 # paths
-PREFIX = /usr/local
+PREFIX = /usr
 MANPREFIX = ${PREFIX}/share/man
 
-X11INC = /usr/X11R6/include
-X11LIB = /usr/X11R6/lib
+X11INC = /usr/include/X11
+X11LIB = /usr/lib
 
 # Xinerama, comment if you don't want it
 XINERAMALIBS  = -lXinerama
